internal/wasmrun/cache/cache.go

```
@@ -0,0 +1,580 @@
+	if err := c.verifyFile(h, path); err == nil {
+		return c.evict()
+	} else if !errors.Is(err, ErrNotFound) && !errors.Is(err, ErrCorrupt) {
+		return err
+	}
+
+	tmp, err := os.CreateTemp(c.dir, filepath.Base(path)+".tmp-")
+	if err != nil {
+		return err
+	}
+	tmpPath := tmp.Name()
+	defer os.Remove(tmpPath)
```

golangci-lint (2.12.2)

[error] Error return value of `os.Remove` is not checked

(errcheck)